person.rb

```diff
@@ -0,0 +1,38 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age, :rentals
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super()
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def can_use_services?
+    return true if of_age?
+
+    false
+  end
+
+  def correct_name
+    name
+  end
+
+  def add_rental(book, date)
+    rental = Rental.new(date, book, self)
+    @rentals.push(rental)
```

Comment:
- [ ] you should remove this since you have  already created an instance of rental class, and remember you already did the push in your rental class initialize :+1: